mms/protocol/otf_message_handler.py

```diff
@@ -252,6 +253,7 @@ def _retrieve_input_data(conn):
     | content_type |
     | input data in bytes |
     """
+    bypass = os.environ.get("MMS_DECODE_INPUT_REQUEST")
```

Comment:
Probably makes sense to rename the variable to match with its purpose: /s/bypass/do_decode

Reply:
Done